mappings/maestro_file_centric.json

```diff
@@ -0,0 +1,224 @@
+            "type": "integer"
+          },
+          "workflow": {
+            "properties": {
+              "name": {
+                "type": "keyword"
+              },
+              "version": {
+                "type": "keyword"
+              }
+            }
+          },
+          "experiment": {
+            "type": "object",
+            "properties": {
+              "platform": {
+                "type": "keyword"
+              },
+              "library_strategy": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      },
+      "file": {
+        "properties": {
+          "id": {
+            "type": "keyword"
+          },
+          "size": {
+            "type": "integer"
+          },
+          "md5sum": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword",
+            "copy_to": ["file_autocomplete"]
+          }
+        }
+      },
+      "index_file": {
+        "properties": {
+          "object_id": {
+            "type": "keyword"
+          },
+          "format": {
+            "type": "keyword"
+          },
+          "md5sum": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
+          },
+          "size": {
+            "type": "integer"
+          }
+        }
+      },
+      "donors": {
+        "type": "nested",
+        "properties": {
+          "id": {
+            "type": "keyword",
+            "copy_to": ["file_autocomplete"]
+          },
+          "submitter_donor_id": {
+            "type": "keyword",
+            "copy_to": ["file_autocomplete"]
+          },
+          "gender": {
+            "type": "keyword"
+          },
+          "specimens": {
+            "type": "nested",
+            "properties": {
+              "id": {
+                "type": "keyword",
+                "copy_to": ["file_autocomplete"]
+              },
+              "submitter_specimen_id": {
+                "type": "keyword",
+                "copy_to": ["file_autocomplete"]
+              },
+              "tumour_normal_designation": {
+                "type": "keyword"
+              },
+              "specimen_tissue_source": {
+                "type": "keyword"
+              },
+              "specimen_type": {
+                "type": "keyword"
+              },
+              "samples": {
+                "type": "nested",
+                "properties": {
+                  "id": {
+                    "type": "keyword",
+                    "copy_to": ["file_autocomplete"]
+                  },
+                  "submitter_sample_id": {
+                    "type": "keyword",
+                    "copy_to": ["file_autocomplete"]
+                  },
+                  "sample_type": {
+                    "type": "keyword"
+                  },
+                  "matched_normal_submitter_sample_id": {
+                    "type": "keyword"
```

Comment:
add `"copy_to": ["file_autocomplete"]`? That should be useful for searching files by matched normal

Reply:
done